docs/guides/vault/create-a-vault.md

```diff
@@ -11,29 +14,205 @@ This guide will teach you how to deploy an **[LSP9Vault](../../standards/smart-c
 
 ## Deploy an LSP9Vault contract
 
-Check **[previous guides](../universal-profile/create-profile#step-1---create-an-eoa)** to learn how to connect to LUKSO's L16 network and load your EOA.
+To start with this guide you will need the following things:
 
-```typescript title="Deploying the vault"
+- The _private key_ of your account, in order to send the transaction.
+- The address of a [**Universal Profile**](../../standards/universal-profile/lsp0-erc725account.md)
+
+Make sure you have the following dependencies installed before beginning this tutorial.
+
+- You can use either [`web3.js`](https://github.com/web3/web3.js) or [`ethers.js`](https://github.com/ethers-io/ethers.js/)
+- You SHOULD install [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```shell title="Install the dependencies"
+npm install web3 @lukso/lsp-smart-contracts
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```shell title="Install the dependencies"
+npm install ethers @lukso/lsp-smart-contracts
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Step 1 - Setup imports and constants​
+
+For the imports, we will need the `LSP9vault` contract **ABI** and **bytecode** in order to deploy a LSP9 Vault.  
+For the contants we will need the _private key_ and _the address of the vault receiver_.
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```typescript title="Imports & Constants"
+import LSP9Vault from '@lukso/lsp-smart-contracts/artifacts/LSP9Vault.json';
 import Web3 from 'web3';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+const vaultReceiver = '0x...'; // The address that will be the vault owner
```

Comment:
IS this the vault receiver or the vault owner? I think the comment makes this confusing and it might be more clear wihtout it